src/middlewares/request.middleware.js

```diff
@@ -295,10 +295,26 @@ function checkChannelID (req, res, next) {
   next()
 }
 
+function addChannelFilters (req, res, next) {
+  // Assuming that the request will not send the filter of channels
+  filterService.getChannelSearchString(req, function (err, channels) {
+    if (err) {
+      LOG.error(utilsService.getLoggerData({}, 'ERROR', filename, 'addChannelFilters',
+        'failed to get channels'))
+    }
+    if (channels && (!_.isEmpty(channels))) {
+      req.body.request.filters.channel = channels
```

Comment:
What happens if request body already has a channel filter? the configuration is overwriting it...